### ISSUE TITLE
Warnings

### ISSUE THREAD

**Issue by r-uben** [facebookresearch/nougat#249]
Hi,

I'm getting the following warnings:

1.  UserWarning: Argument 'alpha_affine' is not valid and will be ignored.
2. UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at [/Users/runner/work/pytorch/pytorch/pytorch/aten/src/ATen/native/TensorShape.cpp:3596](https://file+.vscode-resource.vscode-cdn.net/Users/runner/work/pytorch/pytorch/pytorch/aten/src/ATen/native/TensorShape.cpp:3596).)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
  
 Thanks for your work

**Comment by Marina-mal** [comment]
yes i get the same warnings, when they appear the pdf are not process. But the same script does work some other times. I do not know what is the issue.
